databases/week3/week3.sql

```diff
@@ -0,0 +1,129 @@
+  `title` varchar(255)  NOT NULL,
+  `description` text  NOT NULL,
+  `location` varchar(255)  NOT NULL,
+  `when` datetime  NOT NULL,
+  `max_reservations` int(10)  NOT NULL,
+  `price` decimal  NOT NULL,
+  `created_date` date  NOT NULL
+);
+DROP TABLE IF EXISTS review;
+CREATE TABLE `review` (
+  `id` int(10)  NOT NULL PRIMARY KEY,
+  `title` varchar(255)  NOT NULL,
+  `description` text  NOT NULL,
+  `meal_id` int(10)  NOT NULL,
+  `stars` int(10)  NOT NULL,
+  `created_date` date NOT NULL
+);
+
+ ALTER TABLE `reservation` ADD FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`);
+
+ ALTER TABLE `review` ADD FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`);
+ -- meal table
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values (1, 'chicken pasta','spicy','copenhagen','2020-10-20 12:10:15', '5', '200','2020-10-20');
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values (2, 'fish nuggets','with ketchup','hellerup','2020-09-20 12:12:12','8','80','2020-09-20');
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(3, 'spring rolls','vegetable','ballerup','2020-12-13 14:02:10','6', '50' ,'2020-12-12'); 
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(4, 'burger', 'chicken', 'lyngby', '2020-05-06 11:10:12' ,'5','120','2020-05-06');
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(5, 'sandwitch', 'chicken', 'bagsværd', '2020-10-05 10:02:15', '3', '80', '2020-10-05'); 
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values(6, 'rød grød med flød', 'egg', 'buddinge','2020-08-12 12:05:10' , '2' ,'50', '2020-08-12');
+ 
+ SELECT * FROM meal
+ WHERE id=1;
+
+ UPDATE meal
+ SET title = 'pizza', location='farum'
+ WHERE id=1;
+ SELECT * FROM meal;
+
+  DELETE FROM meal 
+  WHERE id=2;
+  SELECT * FROM meal;
+
+ -- reservation table
+ insert into  reservation(id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values(1, '4','1','2020-10-20', '12345678','alexander','abc@mail.com');
+ insert into  reservation(id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values(2, '6','1','2020-10-20', '12312345','mikell','xyz@mail.com');
+ insert into  reservation(id, number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values(3,'4', '1', '2020-10-21','32456178', 'catherine' ,'svd@mail.com');
+  
+  SELECT * FROM reservation
+  WHERE id=1;
+
+  UPDATE reservation
+  SET contact_name= 'john'
+  WHERE id=1;
+
+ DELETE FROM reservation
+ WHERE id=2;
+
+-- review table
+ insert into  review(id, title, description, meal_id,  stars, created_date) values(1,'pizza', 'good', 1, '4', '2020-10-20'); 
```

Comment:
add some more data in order to execute functionality such as "Sort all meals by average number of stars in the reviews"